src/ralph/accounts/views.py

```diff
@@ -124,7 +127,58 @@ def post(self, request, *args, **kwargs):
         return HttpResponseRedirect(reverse('current_user_info'))
 
 
-class CurrentUserInfoView(UserInfoMixin, RalphBaseTemplateView):
+class _AcceptanceProcessByCurrentUserMixin(object):
+    _config = settings.ACCEPT_ASSETS_FOR_CURRENT_USER_CONFIG
+
+    @property
+    def transition_id(self):
+        return self._config['TRANSITION_ID']
+
+    @property
+    def transition_exists(self):
+        return Transition.objects.filter(id=self.transition_id).exists()
+
+    def post(self, request, *args, **kwargs):
+        assets_to_accept = self.get_assets_to_accept()
+        admin_instance = ralph_site.get_admin_instance_for_model(
+            BackOfficeAsset
+        )
+        url_name = admin_instance.get_transition_bulk_url_name()
+        if assets_to_accept:
+            url = reverse(url_name, args=(self.transition_id,))
+            query = urlencode([('select', a.id) for a in assets_to_accept])
+            return HttpResponseRedirect('?'.join((url, query)))
+        return super().get(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['transition_id'] = self.transition_id
```

Comment:
consider renaming it to sth related to acceptance (ex. `acceptance_transition_id`)